lint/clippy.bzl

```diff
@@ -0,0 +1,185 @@
+"""API for declaring a clippy lint aspect that visits rust_{binary|library|test} rules.
+
+Typical usage:
+
+First, install `rules_rust` into your repository: https://bazelbuild.github.io/rules_rust/. For instance:
```

Comment:
does it require some minimum version of rules_rust to pick up your changes? would be good to document